crates/model/src/orders/mod.rs

```diff
@@ -110,6 +111,8 @@ pub enum OrderError {
     AlreadyInitialized,
     #[error("Order had no previous state")]
     NoPreviousState,
+    #[error("{0}")]
```

Comment:
@cjdsellers this doesn't look right to me, but used this approach to make it pass tests